IPython/html/static/base/js/keyboard.js

```diff
@@ -250,7 +250,10 @@ define([
             if (a.help_index < b.help_index){
                 return -1;
             }
-            return 0;
+			if (a.help_index = b.help_index){
```

Comment:
This doesn't look right. We've already checked `a.help_index > b.help_index` and `a.help_index < b.help_index` above, if it's reached this point then they must be equal.

(Also `=` assigns the RHS to the LHS - you'd want `===` to check if they're the same)